tests/integrations/raftstore/test_lease_read.rs

```diff
@@ -312,3 +312,23 @@ fn test_node_callback_when_destroyed() {
         resp
     );
 }
+
+/// Test if the callback proposed by read index is cleared correctly.
+#[test]
+fn test_lease_read_callback_destroy() {
```

Comment:
Does it fail without your change?

Reply:
Yes, it will panic at https://github.com/tikv/tikv/pull/4682/files#diff-79ba1b5dd0906a82a30f8ab9eba6ebd2R853.